### PR TITLE
Feature efs media

### PR DESCRIPTION
**Pull request by pinkeen**

### CODE REVIEW

File: roles/cs.aws-cloudfront/templates/s3-media-distribution-config.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
                         ],
                         "Quantity": 3
                     },
-                    "OriginProtocolPolicy": "http-only",
+                    "OriginProtocolPolicy": "https-only",
```
Comment (krisdante) [inline roles/cs.aws-cloudfront/templates/s3-media-distribution-config.json]: We support HTTP for static files. No need to engage encryption here.

**Comment by pinkeen** [comment]
Please do not merge yet, there seems to be a problem with CORS/preflight after changing the static files origin to the website/LB.

File: roles/cs.aws-cloudfront/templates/s3-media-distribution-config.json (hunk 1 above)
Comment (pinkeen) [reply roles/cs.aws-cloudfront/templates/s3-media-distribution-config.json]: reverted

**Comment by pinkeen** [comment]
CORS problem fixed via 1594d30 - can be merged now

**Pull request merged**